Client/Frontend/Shields/SimpleShieldsView.swift

```diff
@@ -39,16 +39,39 @@ class SimpleShieldsView: UIView, Themeable {
     // Shields Up
     
     class BlockCountView: UIView, Themeable {
+        
+        private struct UX {
+            static let descriptionEdgeInset = UIEdgeInsets(top: 13, left: 16, bottom: 14, right: 16)
```

Comment:
Is it 13 on top and 14 on bottom? Might as well make both 14.

Reply:
Fix at https://github.com/brave/brave-ios/pull/3387/commits/da11867eb41296b14bf62ec79fb7344b5577df37